content/id/docs/concepts/workloads/controllers/daemonset.md

```diff
@@ -0,0 +1,234 @@
+---
+title: DaemonSet
+content_template: templates/concept
+weight: 50
+---
+
+{{% capture overview %}}
+
+DaemonSet memastikan semua atau sebagian Node memiliki salinan sebuah Pod.
+Ketika node baru ditambahkan ke klaster, Pod ditambahkan ke node tersebut.
```

Comment:
since `node` is a kubernetes object, we need to make this capital `Node`, same comment for the rest of the word on this page